<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FanSettings</class>
 <widget class="QWidget" name="FanSettings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>300</width>
    <height>159</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QFrame {
	border: 2px solid black;
	border-radius: 4px;
	padding: 2px;
}

QComboBox {
	selection-background-color: transparent;
	selection-color: black;
}

QPushButton {
	border: 2px solid #8f8f91;
	border-radius: 6px;
	background-image: url(:/Hac/background-button-big.png);
	background-repeat: no-repeat;
	width: 48px;
	height: 17px;
 }

 QPushButton:pressed {
	border-color: red;
 }

QSpinBox {
	margin: 1px;
	background: transparent;
	selection-background-color: transparent;
	text: black;
	selection-color: black;
	font: normal 15px;
	border: 1px solid black;
}

QSpinBox::up-button {
	subcontrol-origin: border;
    subcontrol-position: left;
	background: transparent;
	width: 20px;
	height: 20px;
}

QSpinBox:up-arrow {
     image: url(:/Hac/leftarrow-black-20x20.png);
     width: 20px;
     height: 20px;
}
QSpinBox::up-arrow:pressed {
     image: url(:/Hac/leftarrow-red-20x20.png);
}


QSpinBox::down-button {
    subcontrol-origin: border;
    subcontrol-position: right;
	background: transparent;
	width: 20px;
	height: 20px;
}

QSpinBox::down-arrow {
     image: url(:/Hac/rightarrow-black-20x20.png);
     width: 20px;
     height: 20px;
}
QSpinBox::down-arrow:pressed {
     image: url(:/Hac/rightarrow-red-20x20.png);
}


</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="2">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>49</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="0">
    <spacer name="horizontalSpacer_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>90</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="1" colspan="2">
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="unitLabel">
       <property name="text">
        <string>Temperature Unit:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QComboBox" name="unitComboBox">
       <property name="maxCount">
        <number>100</number>
       </property>
       <property name="sizeAdjustPolicy">
        <enum>QComboBox::AdjustToContentsOnFirstShow</enum>
       </property>
       <property name="frame">
        <bool>false</bool>
       </property>
       <item>
        <property name="text">
         <string>Celsius (℃)</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Fahrenheit (ºF)</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="rangeLabel">
       <property name="text">
        <string>Temperature Range:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QComboBox" name="minComboBox">
         <property name="maximumSize">
          <size>
           <width>50</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="frame">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string> To</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="maxComboBox">
         <property name="maximumSize">
          <size>
           <width>50</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="frame">
          <bool>false</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item row="1" column="3">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="1">
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>49</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="3" column="0" colspan="3">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <spacer name="horizontalSpacer_4">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="cancelButton">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="okButton">
       <property name="text">
        <string>Ok</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
